backend/coins/btc/transaction_test.go

```diff
@@ -269,10 +287,56 @@ func TestPickChangeAddressFails(t *testing.T) {
 		SigningConfigurations: signingConfiguration,
 	})
 	require.NoError(t, account.Initialize())
-	account.ensureAddresses()
 	require.Eventually(t, account.Synced, time.Second, time.Millisecond*200)
 
 	account.subaccounts = subaccounts{}
 	_, err := account.pickChangeAddress(nil)
 	require.ErrorContains(t, err, "Account has no subaccounts")
 }
+
+func TestTxProposal(t *testing.T) {
+	testCases := []struct {
```

Comment:
would be nice to also test:

- SpendAll (one of the two major modes of spending)
- SelectedUTXOs for coin control, with and without coin control
- that amount parsing works as expected if sats mode is enabled (user enters the amount in satohis instead of BTC)

Reply:
Done!

Reply:
> SelectedUTXOs for coin control, with and without coin control

Though I'm not sure what you mean "with and without coin control". Isn't specifying SelectedUTXOs in the args efffectively "coin control"?

Reply:
Yeah, it's the same.

By with and without coin control I meant: there should be a test using SelectedUTXOs with SendAll, and one using SelectedUTXOs with a regular amount (not send all). The first is now there, the second is still missing I think.

Reply:
I see what you mean now, done!